index.js

```diff
@@ -1,6 +1,6 @@
 const utils = require('./helpers/utils');
 
-const desserts = ['akok', 'apple crumble', 'cake', 'cempedak creme', 'cempedak fritters', 'cekodok','cendol','ais kacang','eclair','chocolate','cake','tart', 'muffin', 'brownies', 'cupcake', 'frgál', 'kremrole', 'větrník', 'buchty', 'krapfen', 'kek lapis']
+const desserts = ['ais kacang', 'akok', 'apple crumble', 'brownies', 'buchty', 'cake', 'cekodok', 'cempedak creme', 'cempedak fritters', 'cendol', 'chocolate', 'cupcake', 'eclair', 'frgál', 'kek lapis', 'krapfen', 'kremrole', 'muffin', 'tart', 'větrník']
```

Comment:
What's a buchty?

Reply:
existing entry mate, I have no idea.

Reply:
Ah yes, these are actually pretty delicious.
https://en.wikipedia.org/wiki/Buchteln

Reply:
This was added by you actually.
https://github.com/sanijalal/random-dessert-generator/pull/3/files

Reply:
lol, I don't remember. Thanks for the refresher.